server/tso/allocator_manager.go

```diff
@@ -1164,3 +1187,8 @@ func (am *AllocatorManager) nextLeaderKey(dcLocation string) string {
 func (am *AllocatorManager) EnableLocalTSO() bool {
 	return am.enableLocalTSO
 }
+
+// SetEnableLocalTSO set the value of AllocatorManager.enableLocalTSO.
+func (am *AllocatorManager) SetEnableLocalTSO(enableLocalTSO bool) {
```

Comment:
Why do we need this function? After `server.SetEnableLocalTSO(false)` and rebooting, `tso.NewAllocatorManager` will have the false value automaticlly.